### PR TITLE
`v9.0.0` release

### PR DESCRIPTION
**Pull request by Mati365**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Changelog for `v9.0.0`.

### CODE REVIEW

**Comment by Mati365** [comment]
@pomek Should we include changelog from alpha? 

Needs to be released after @Witoso approval

**Comment by Mati365** [comment]
Ok :+1: I'd try downgrade.

**Comment by Mati365** [comment]
@Witoso Downgraded, looks ok.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+## [9.0.0](https://github.com/ckeditor/ckeditor5-angular/compare/v9.0.0-alpha.0...v9.0.0) (2024-09-19)
+
+### BREAKING CHANGES
+
+* Changed minimum supported version of Angular to the oldest supported LTS: `>=16.x`.
+* Switched compiler from TypeScript `4.x` to `5.x` to ensure compatibility with the newest CKEditor 5 packages.
+
+### Features
+
+* Add cloud CDN support. ([commit](https://github.com/ckeditor/ckeditor5-angular/commit/9c62280f4bdc487ccc258437ee7b6b7cbfa8cbf0))
```
Comment (Witoso) [inline CHANGELOG.md:13]: Please mention it's experimental.

Comment (Mati365) [reply CHANGELOG.md]: Done

**Comment by Witoso** [comment]
We have it pinned in the editor, not sure why:

> typescript": "5.0.4",

@filipsobol?

**Comment by Mati365** [comment]
@Witoso It's due to `~` prefix in `typescript` package version. It uses the newest patch version.

**Comment by Witoso** [comment]
I know, I just wonder should we pin it exactly like `ckeditor5`, not sure.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,18 @@
 Changelog
 =========
 
+## [9.0.0](https://github.com/ckeditor/ckeditor5-angular/compare/v9.0.0-alpha.0...v9.0.0) (2024-09-19)
+
+### BREAKING CHANGES
+
+* Changed minimum supported version of Angular to the oldest supported LTS: `>=16.x`.
```
Comment (pomek) [inline CHANGELOG.md:8]: We need to update several files, too:

* https://github.com/ckeditor/ckeditor5-angular#supported-angular-versions
* https://ckeditor.com/docs/ckeditor5/latest/getting-started/installation/angular.html#supported-angular-versions

In the past, we tracked a package version and a compatible Angular version. I would leave the decision to you (a team) whether to continue following this rule or completely remove it.

**Comment by pomek** [comment]
Once the PR is merged, please ping @psmyrek / @martnpaneq to process the publishing process.

File: README.md [hunk 3]
```diff
@@ -35,12 +35,17 @@ Because of the breaking changes in the Angular library output format, the `@cked
       <td colspan="3">Actively supported versions</td>
     </tr>
     <tr>
-      <td><code>^7</code></td>
-      <td><code>13+</code></td>
-      <td>Changes in peer dependencies. (<a href="https://github.com/ckeditor/ckeditor5-angular/issues/376">issue</a>)</td>
```
Comment (pomek) [inline README.md:40]: Now, we miss `7` in the table :grin:

image

**Review by pomek — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (Mati365) [reply CHANGELOG.md:8]: done https://github.com/ckeditor/ckeditor5/pull/17121

**Pull request merged**